cart.js

```diff
@@ -0,0 +1,120 @@
+let products=[{  
+    id:1,
+    name: "Perfume",
+    price: "$100",
+    category:"Cosmetics",
+    imageURL:"images/perfume.jpg" ,
+        },{
+    id:2,
+    name: "Lipstick",
+    price: "$25",
+    category:"Cosmetics",
+    imageURL:"images/lipstick.jpg" ,
+        },{
+    id:3,
+    name: "Apple Airpods",
+    price: "$100",
+    category:"Technology",
+    imageURL:"images/airpods.jpg" ,
+        }];
+
+let cards = document.createElement("div");
+let cardsContainer = document.getElementsByClassName("cards-container")[0];
+cardsContainer.appendChild(cards);
+
+products.map(function(product){
```

Comment:
Since this map() isn't returning anything, I would suggest changing it into a forEach instead.  It will work the same, but its intent will be a little clearer then.